src/Seo.php

```diff
@@ -246,4 +246,16 @@ public static function adjustPrice($markup, $product) {
     return $markup;
   }
 
+  /**
+   * Fixes schema.org product availability.
+   *
+   * @implements woocommerce_structured_data_product_offer
+   */
+  public static function adjustAvailability($markup, $product) {
+    if ($product->get_stock_quantity() || $product->backorders_allowed()) {
+      $markup['availability'] = 'http://schema.org/InStock';
```

Comment:
`http://schema.org/InStock` is redirecting to `https://schema.org/InStock`.
Is not better to use the SSL protected version in our code?